src/graph.js

```diff
@@ -1,8 +1,53 @@
 /* eslint-disable */
 class Graph {
-  constructor() {
-    this.graph = {};
+  constructor() {}
```

Comment:
Huh, interesting implementation and idea to just use the `this` object for storing the graph nodes. I've never seen it done this way before.